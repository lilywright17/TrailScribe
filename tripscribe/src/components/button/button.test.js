// snapshot test of our button component

import React from 'react';
import Button from './Button';
import renderer from 'react-test-renderer';

test('renders button component correctly with buttonText passed by props', () => {
    const buttonText = "Back";
    const component = renderer.create(<Button buttonText={buttonText} />)
    const tree = component.toJSON();
    
    // creates snapshot
    expect(tree).toMatchSnapshot();
    })
