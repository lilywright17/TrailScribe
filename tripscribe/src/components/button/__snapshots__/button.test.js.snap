// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button component correctly with buttonText passed by props 1`] = `
<div>
  <button
    className="buttonPrimary"
  />
</div>
`;
